### PR TITLE
removed styles that blocked inputs to support type number, removed st…

### LINKED ISSUE #4312: Input type=“number” isn't supported
**Description**:
Input type=“number” isn't supported

**Expected behavior**:
input type=“number” is supported

**To Reproduce**:
1. Open PDP 
2. Add product to cart
3. Open Shipping step
4. Open Console
5. Change input type to type=“number”

**Actual result:**:
input type=“number” isn't supported
Screenshot 2022-02-18 at 11 22 25

### PR DESCRIPTION
**Pull request by AzizKHAN030**
…yles added for the custom FiledNumber components

**Related issue(s):**
* Fixes #4312 

**Problem:**
* Input fields didn't support type number

**In this PR:**
* Removed styles that blocked fields to support number type and put that styles in a separate file because they styled custom FieldNumber Component

### CODE REVIEW

**Comment by zans-laksa** [comment]
Please make your pull requests correctly. Your title starts in the PR title but ends in the description.

**Review by zans-laksa — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/FieldNumber/FieldNumber.style.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            cursor: default;
+            pointer-events: none;
+        }
+
+        span {
+            @include mobile {
+                inset-block-start: -2px;
+            }
+        }
+    }
```
[the remark this answers is not in the document]
Comment (AzizKHAN030) [reply packages/scandipwa/src/component/FieldNumber/FieldNumber.style.scss:76]: All the styles were written before in the _input.scss file, and the selector was for type number, consequently it restyled number inputs completely. Task was either use our own FieldNumber components for increasing/decreasing item quantity and either make ordinary inputs support default number appearance (e.g for telephone numbers). Noticed that FieldNumber component has constant class part "item_qty", all I made is that removed styles from global _input.scss file and created styles file for FieldNumber component and added all the styles in the new file.  Selector is if the element has type number and "item_qty" part in classname.

[the remark this answers is not in the document]
Comment (zans-laksa) [reply packages/scandipwa/src/component/FieldNumber/FieldNumber.style.scss:76]: The previous implementation was done in the wrong way. You cannot rely on it.

[the remark this answers is not in the document]
Comment (zans-laksa) [reply packages/scandipwa/src/component/FieldNumber/FieldNumber.style.scss:76]: The number field should never be used for phone numbers.
There is the `type="tel"` for such cases.

[the remark this answers is not in the document]
Comment (zans-laksa) [reply packages/scandipwa/src/component/FieldNumber/FieldNumber.style.scss:76]: > They wanted for number types to have regular appearance

This isn't related to scandipwa at all.

We should support regular number fields and number fields with quantity controls.
Thus we should either make this component configurable or split it into two components.
But it should be discussed first.

**Pull request closed without merging**